### PR TITLE
Flo

### PR DESCRIPTION
**Pull request by haslbeck**
@cfuselli 
 I remember you had your import_data.py running on your machine - any chance this was python3? 
I would either need to check this with on tuesday with a detector connected or you tomorrow on your machine...
So either you pull from my branch and try before merge or we wait. you decide:)

@alisidley @cfuselli 
In general: I suggest a discussion which python version to use... The legacy stuff is written in python2 but I would imagine that all people who use any "fancy" editor like PyCharm or VSCode run python3 so we better check for compatability.
-> added TODISCUSS.md file for questions that need a discussion.

-fh

### CODE REVIEW

File: Recording_Data/import_data.py [hunk 1]
```diff
@@ -17,8 +17,8 @@
 import multiprocessing
 import math
 import random
-import thread
-import serial
```
Comment (haslbeck) [inline Recording_Data/import_data.py:21]: double in the legacey?

**Comment by cfuselli** [comment]
I would wait for everyone to do the first test pull request first

**Comment by haslbeck** [comment]
@cfuselli  sure! lets wait for everyone.

**Comment by haslbeck** [comment]
- making a new one for clearness -fh

**Pull request closed without merging**